js/id/renderer/mapillary_sign_layer.js

```diff
@@ -0,0 +1,177 @@
+iD.MapillarySignLayer = function(context) {
+    var roundedProjection = iD.svg.RoundProjection(context.projection),
+        urlSearch = 'https://a.mapillary.com/v2/search/im/geojson/or',
+        urlImage = 'http://mapillary.com/map/im/',
+        urlThumb = 'https://d1cuyjsrcm0gby.cloudfront.net/',
+        clientId = 'NzNRM2otQkR2SHJzaXJmNmdQWVQ0dzoxNjQ3MDY4ZTUxY2QzNGI2',
```

Comment:
please change to 
    clientId='NzNRM2otQkR2SHJzaXJmNmdQWVQ0dzo1ZWYyMmYwNjdmNDdlNmVi'